app.json

```diff
@@ -0,0 +1,31 @@
+{
+    "name": "F.Lute",
+    "description": "DSP as a Discord Bot",
+    "keywords": [
+      "Entertainment",
+      "python",
+      "bot"
+    ],
+    "repository": "https://github.com/ClarityCafe/F.Lute",
+    "logo": "https://cdn.discordapp.com/avatars/436026947974463488/7eb55b489b3335e1bf78891bc3b38ae1.png?size=512",
+    "env": {
+      "FLUTE_PREFIX": {
+        "description": "Prefix for the bot",
+        "generator": "flute "
```

Comment:
Prefix is a list, not a string. See the config.example for the default value.

Reply:
I don't think we can do lists in environment variables.

Reply:
Just split the string by comma or something in the python code.